js/modules/loki_app_dot_net_api.js

```diff
@@ -324,22 +407,86 @@ class LokiAppDotNetServerAPI {
         fetchOptions.body = rawBody;
       }
       fetchOptions.headers = new Headers(headers);
-      result = await nodeFetch(url, fetchOptions || undefined);
     } catch (e) {
-      log.info(`e ${e}`);
+      log.info('serverRequest set up error:', JSON.stringify(e));
       return {
         err: e,
       };
     }
+
+    let result;
     let response = null;
-    try {
-      response = await result.json();
-    } catch (e) {
-      log.warn(`serverRequest json parse ${e}`);
-      return {
-        err: e,
-        statusCode: result.status,
-      };
+    let TxtResponse = '';
```

Comment:
Variables everywhere else in our code start with a lower-case letter. Any reason this is capitalised?

Reply:
none, will fix